sources/Common/src/Log.cpp

```diff
@@ -15,20 +15,28 @@
 
 #include "Log.h"
 
+#include <boost/filesystem.hpp>
+
+namespace file = boost::filesystem;
+
 namespace dfl {
 namespace common {
 
 const char* Log::dynaflowLauncherLogTag = "DYNAFLOW_LAUNCHER";
 
 void
-Log::init(const common::Options& options) {
+Log::init(const common::Options& options, const std::string& outputDir) {
   using DYN::Trace;
   auto& config = options.config();
 
   std::vector<Trace::TraceAppender> appenders;
   Trace::TraceAppender appender;
+  file::path outputPath(outputDir);
+  if (!file::exists(outputPath)) {
```

Comment:
maybe remove the corresponding lines in context.exportOuputs

Reply:
done